regression/tests/helpers.py

```diff
@@ -90,3 +95,100 @@ def login(login_page):
         login_email = LoginHelper.get_login_email()
         login_password = LoginHelper.get_login_password()
         login_page.login(login_email, login_password)
+
+
+class LoginApi(object):
+    """
+    An Api to login the stage.
+    """
+    def __init__(self):
+        self.login_url = 'https://courses.stage.edx.org/login'
+        self.session = requests.Session()
+        self.session.auth = (
+            BASIC_AUTH_USERNAME,
+            BASIC_AUTH_PASSWORD
+        )
+        self.login_response = None
+
+    def check_response(self, response):
+        """
+        Check whether a response was successful. If not raise an exception
+
+        Arguments:
+            response: HTTP response object
+        """
+        if response.status_code != 200:
+            raise Exception(
+                'API request failed with following error code: ' +
+                str(response.status_code)
+            )
+
+    def post_headers(self, x_csrf):
+        """
+        Header which are to be used in the POST Requests
+
+        Arguments:
+            x_csrf: Cross site request forgery protection token.
+        """
+        return {
+            'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8',
+            'Accept': '*/*',
+            'X-Requested-With': 'XMLHttpRequest',
+            'Referer': self.login_url,
+            'X-CSRFToken': x_csrf,
+        }
+
+    def create_base_session(self):
+        """
+        Create a session with the host.
+        """
+        response = self.session.get(self.login_url)
+        self.check_response(response)
+        self.session.cookies = response.cookies
+        self.session.headers = self.post_headers(
+            response.cookies['csrftoken']
+        )
+
+    def login(self):
+        """
+        Login to the stage.
+        """
+        self.create_base_session()
+        login_post_url = 'https://courses.stage.edx.org/' \
+                         'user_api/v1/account/login_session/'
+        payload = {
+            'email': LOGIN_EMAIL,
+            'password': LOGIN_PASSWORD,
+            'remember': 'false'
+        }
+        response = self.session.post(login_post_url, data=payload)
+        self.check_response(response)
+        self.session.cookies = response.cookies
+        self.session.headers = self.post_headers(
+            response.cookies['csrftoken']
+        )
+        self.login_response = response
+
+    def authenticate(self, browser):
+        """
+        Authenticate the user and pass the session to the browser.
+
+        Arguments:
+        browser: Browser to pass the session to.
+        """
+        self.login()
+        # To make cookies effective, we have to set the
+        # domain of the browser the same as that of the
+        # cookies. To do this, just visit a page of the
+        # same domain.
+        # Cookies require the domain to be ".stage.edx.org"
+        browser.get(LOGIN_BASE_URL + '/dashboard')
```

Comment:
Leave an additional comment here that this will cause the browser to navigate to the login screen, but its form does not need to be filled out and submitted. Navigation to the next page will result in the browser being logged in.